fastplotlib/layouts/_subplot.py

```diff
@@ -27,12 +18,11 @@ def __init__(
         parent: Any = None,
         position: Tuple[int, int] = None,
         parent_dims: Tuple[int, int] = None,
-        camera: str = "2d",
-        controller: Union[PanZoomController, OrbitController] = None,
-        canvas: Union[str, WgpuCanvas, Texture] = None,
-        renderer: WgpuRenderer = None,
+        camera: Union[str, pygfx.PerspectiveCamera] = "2d",
+        controller: Union[str, pygfx.Controller] = None,
+        canvas: Union[str, WgpuCanvas, pygfx.Texture] = None,
```

Comment:
how can a texture be a canvas?

Reply:
See docstring for the `pygfx.WgpuRenderer`: https://pygfx.readthedocs.io/en/stable/_autosummary/renderers/pygfx.renderers.WgpuRenderer.html#pygfx.renderers.WgpuRenderer

Basically you can render a rasterized scene to a Texture :smile: . A texture is just a way to represent a 2D image. I think that's part of how the remote frame buffer works, _the canvas is a texture_ .